admin/src/app/foms/fom-add-edit/fom-add-edit.component.ts

```diff
@@ -51,10 +54,15 @@ type ApplicationPageType = 'create' | 'edit';
     providers: [DatePipe]
 })
 export class FomAddEditComponent implements OnInit, AfterViewInit, OnDestroy {
+  readonly projectPlanCodeEnum = ProjectPlanCodeEnum;
   fg: RxFormGroup;
   state: ApplicationPageType;
   originalProjectResponse: ProjectResponse;
   districts: DistrictResponse[] = this.stateSvc.getCodeTable('district');
+  projectPlanOptions: ICodeTable[] = [
+    {"code": this.projectPlanCodeEnum.Fsp, "description": "Forest Stewardship Plan"},
+    {"code": this.projectPlanCodeEnum.Woodlot, "description": "Woodlot License Plan"}
+  ];
```

Comment:
I did not follow previous of retrieving code table using "state.service.ts"; feel it is too confusing and not necessary. Direct using generated api-client enum and only hardcode description for this code.

Reply:
The purpose of state.service.ts is to pre-load all code tables from backend and cache the values so that they aren't being looked up each time in the UI via another backend call. For projectPlanCode, if it is a hardcoded list here then I agree you don't need to cache in state.service.ts